src/components/molecules/NavBar/NavBar.scss

```diff
@@ -269,8 +270,8 @@ $border-radius: 28px;
     height: 20px;
     width: 20px;
     background-image: url(/icons/nav-party-logo-icon.png);
+    filter: brightness(0);
     background-size: 20px;
-    opacity: 0.4;
```

Comment:
Presumably the same reasoning as https://github.com/sparkletown/sparkle/pull/1240/files#r588871134